packages/world/src/modules/core/implementations/AccessManagementSystem.sol

```diff
@@ -59,4 +59,20 @@ contract AccessManagementSystem is System {
     // Grant access to new owner
     ResourceAccess._set(namespaceId, newOwner, true);
   }
+
+  /**
+   * @notice Renounces ownership of the given namespace
+   * @dev Requires the caller to own the namespace. Revoke ResourceAccess for previous owner
+   * @param namespaceId The ID of the namespace to transfer ownership.
+   */
+  function renounceOwnership(ResourceId namespaceId) public virtual {
```

Comment:
need to make sure we add a `requireNamespace` check here once https://github.com/latticexyz/mud/pull/2142 lands